### PR TITLE
docs: change_for_airflow_provider_demo_of_integration_folder

### PR DESCRIPTION
**Pull request by TanZiYen**

### CODE REVIEW

**Review by Elliezza — APPROVED** [review]

File: docs/en/integration/deploy_integration/airflow_provider_demo.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+The diagram above illustrates the work process in DAG. It begins by creating a table, followed by offline data loading, feature extraction, and model training. If the model trained performs well (AUC >= 99.0), the workflow proceeds to execute deploy SQL and model serving online. Otherwise, a failure report is generated.
+
+In the following demonstration, you can directly import this DAG and run in Airflow.
+
+
+## Demonstration
+
+We import the above DAG to perform feature computation and deployment for the TalkingData Demo, then perform real-time inference using the predict server of TalkingData Demo.
+
+### 0 Preparation
```
Comment (lumianph) [inline docs/en/integration/deploy_integration/airflow_provider_demo.md]: remove the numbering, can change it to "Step 1: xxx" 
(change other places in this doc and the Chinese doc accordingly)

File: docs/en/integration/deploy_integration/airflow_provider_demo.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+#### 0.1 Download DAG
```
Comment (lumianph) [inline docs/en/integration/deploy_integration/airflow_provider_demo.md]: remove the numbering, 0.1/0.2/...

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/en/integration/deploy_integration/airflow_provider_demo.md (hunk 1 above)
Comment (lumianph) [reply docs/en/integration/deploy_integration/airflow_provider_demo.md]: @Elliezza

Comment (Elliezza) [reply docs/en/integration/deploy_integration/airflow_provider_demo.md]: updated, please advise @lumianph

Comment (lumianph) [reply docs/en/integration/deploy_integration/airflow_provider_demo.md]: no... i mean change the third-level titles to Step XXX (Preparation, Start OpenMLDB and Airflow, Running DAG, Test), then remove all other numbers in the titles.
we do not use the numbers in titles for most docs (except a few that also need to be changed later)

Comment (Elliezza) [reply docs/en/integration/deploy_integration/airflow_provider_demo.md]: updated, please check @lumianph

**Review by lumianph — APPROVED** [review]

**Pull request merged**